apps/api/src/app/chat/chat.module.ts

```diff
@@ -5,7 +5,7 @@ import { AgentsModule } from './agents/agents.module';
 import { ChatSockets } from './chat.sockets';
 
 @Module({
-  imports: [AgentsModule, AssistantModule.forRoot(assistantConfig)],
+  imports: [AgentsModule, AssistantModule.register(assistantConfig)],
```

Comment:
Could you update the `readme` file:
* `forRoot` -> `register`
* new property in the configuration